cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,20 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | Postgre | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling & Repair| Yes | No | No |  |  | No | Yes 
+Synchronous Replication | Yes | No | No |  |  | No |  
+Automatic Failover | Yes | No | No |  |  |  | Yes 
+Read from Any Replica | No |  |  |  |  | Yes |  
```

Comment:
Thanks for those estimations. I'll put them in now.

For "Read from any Replica", @bdarnell mentioned to me that it's possible at the kv layer but not through sql. Is that what you mean here? If so, since we're not encouraging kv access (is it even possible?), shouldn't we mark this "No", maybe with a caveat that it'll be possible later?